Specs/Scene/ShadowMapSpec.js

```diff
@@ -941,6 +941,48 @@ describe(
       });
     });
 
+    it("disable shadows fading", function () {
```

Comment:
It looks like `effectiveDarkness` only gets used in the specs.. I think this test can be rewritten similarly to the `"changes darkness"` spec above. 

Something like:

```
...
scene.shadowMap = new ShadowMap( { ... } );

// Render with light looking straight down
var shadowedColor = renderAndReadPixels();

// Move the light close to the horizon
lightCamera.lookAt(center, new Cartesian3(1.0, 0.0, 0.01));

// Render with faded shadows
var horizonShadowedColor = renderAndReadPixels();
expect(horizonShadowedColor).not.toEqual(shadowedColor);

// Render with unfaded shadows
scene.shadowMap.fadingEnabled = false;
renderAndCall(function (rgba) {
  expect(rgba).not.toEqual(horizonShadowedColor);
});
```

Then the `effectiveDarkness` getter can be removed from `ShadowMap`.

Reply:
thanks @ebogo1 for you suggestions. I made the changes you suggested: removed the `effectiveDarkness` getter and updated the unit test. Let me know if I need to change anything else